### ISSUE TITLE
Monitor input status

### ISSUE THREAD

**Issue by julian-weinert** [justinlatimer/node-midi#98]
I'd like to know if my input port is open already or not. I my UI I have a device selector. When selecting a different device, I want to close the port and reopen it on the corresponding channel.

The problem is, when calling `input.closePort();` whilst nothing is open crashes the script:

```
function connect() {
    input.closePort();
    input.openPort(command.device);
}
```

Ideally I'd like to do something like

```
if (input.isPortOpen()) {
    input.closePort();
}
```

**Comment by julian-weinert** [comment]
Also surrounding the call with try doesn't do the trick, it's still crashing. Might be interesting in the context of https://github.com/justinlatimer/node-midi/issues/97?

**Comment by nroadley** [comment]
I needed this check too. It seemed RTMIDI already had in it. Now i just wish I could know exactly which port is open, to avoid conflicts of having more ports open to the same device.

Made a PR here: #101

**Comment by justinlatimer** [comment]
Fixed by #101
